packages/wrangler/templates/pages-template-worker.ts

```diff
@@ -149,3 +150,9 @@ export default {
     }
   },
 };
+
+const unlockResponse = (response: Response) =>
```

Comment:
BIKESHED - unlock seems like a strange description for this function.

Reply:
I accept your bike shed, haha. Renamed to `cloneResponse` which is what it actually does. The mutability is a desired side effect of this.